extras/wasm-browser-run/src/webdriver.rs

```diff
@@ -138,14 +138,14 @@ pub enum WebdriverKind {
 }
 
 impl WebdriverKind {
-    const CHROMIUM_MAJOR_VERSION: &str = "115";
+    const CHROMIUM_MAJOR_VERSION: &'static str = "115";
```

Comment:
Can we use the occasion to bump this to chromium 119?

 ```suggestion
    const CHROMIUM_MAJOR_VERSION: &'static str = "119";
```